ksqldb-serde/src/main/java/io/confluent/ksql/serde/protobuf/ProtobufFormat.java

```diff
@@ -16,17 +16,24 @@
 package io.confluent.ksql.serde.protobuf;
 
 import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
 import io.confluent.connect.protobuf.ProtobufData;
 import io.confluent.connect.protobuf.ProtobufDataConfig;
 import io.confluent.kafka.schemaregistry.ParsedSchema;
 import io.confluent.kafka.schemaregistry.protobuf.ProtobufSchema;
 import io.confluent.ksql.serde.FormatInfo;
 import io.confluent.ksql.serde.KsqlSerdeFactory;
+import io.confluent.ksql.serde.SerdeFeature;
 import io.confluent.ksql.serde.connect.ConnectFormat;
+import java.util.Set;
 import org.apache.kafka.connect.data.Schema;
 
 public class ProtobufFormat extends ConnectFormat {
 
+  private static final Set<SerdeFeature> SUPPORTED_FEATURES = ImmutableSet.of(
+      SerdeFeature.WRAP_SINGLES
```

Comment:
How come ProtobufFormat has `WRAP_SINGLES` in its set of supported features, but the Kafka and delimited formats have the empty supported feature sets? Seems inconsistent in that I'd expect Kafka and delimited to have `UNWRAP_SINGLES` in their supported feature sets to parallel this.

Reply:
The next PR will add `UNWRAP_SINGLES` for `KAFKA` and `DELIMITED`, it's just not possible right now due to some other issues that are best kept to their own PR.